src/DssSpell.t.sol

```diff
@@ -2331,13 +2333,43 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(dai.balanceOf(PE_MULTISIG)     - prevDaiPe, amountPe * WAD);
         assertEq(dai.balanceOf(ORA_MULTISIG)    - prevDaiOra, amountOra * WAD);
         assertEq(dai.balanceOf(ORA_ER_MULTISIG) - prevDaiOraEr, amountOraEr * WAD);
+    }
 
+    function test_core_unit_budgets_do_not_distribute() public {
+        uint256 prevSin      = vat.sin(address(vow));
+        uint256 prevDaiGro   = dai.balanceOf(GRO_MULTISIG);
+        uint256 prevDaiMkt   = dai.balanceOf(MKT_MULTISIG);
+        uint256 prevDaiGov   = dai.balanceOf(GOV_MULTISIG);
+        uint256 prevDaiRwf   = dai.balanceOf(RWF_MULTISIG);
+        uint256 prevDaiRisk  = dai.balanceOf(RISK_CU_EOA);
+        uint256 prevDaiPe    = dai.balanceOf(PE_MULTISIG);
+        uint256 prevDaiOra   = dai.balanceOf(ORA_MULTISIG);
+        uint256 prevDaiOraEr = dai.balanceOf(ORA_ER_MULTISIG);
+
```

Comment:
can we validate/show here that `done` is true?

Reply:
Done.